doc/conf.py

```diff
@@ -101,10 +102,14 @@
 # The version info for the project you're documenting, acts as replacement for
 # |version| and |release|, also used in various other places throughout the
 # built documents.
-#
-# The short X.Y version.
-version = ''
-# The full version, including alpha/beta/rc tags.
+
+# Latest release version
+version = re.match(
+    r'v?([0-9]+.[0-9]+.[0-9]+).*',
+    os.popen('git describe --tags').read().strip()
+).groups()[0]
+
+# The full current version, including alpha/beta/rc tags.
 import nilearn
 release = nilearn.__version__
```

Comment:
I think during a release, `version` and `release` will be the same. We need to be careful in how we are defining and using these and maybe also use better naming.

Reply:
Yea I agree with the inversion and also why not just be very explicit. 

It's a problem if we do something like the first point I propose in the previous comment:
> - Add an else statement to define the same thing but inverted for the stable docs

because during the release process the banner for the stable docs would be created with a reference to the same version number for both `dev` and `stable`. But I think we can solve that in an automated way with some additional code.